package.json

```diff
@@ -105,12 +105,10 @@
   "lint-staged": {
     "*.js": [
       "eslint --fix",
-      "prettier --write --ignore-path .eslintignore",
-      "git add"
+      "prettier --write --ignore-path .eslintignore"
     ],
     "*.md": [
-      "prettier --write --ignore-path .eslintignore",
-      "git add"
```

Comment:
is git add no longer needed with lint-staged v10? @daKmoR I think you know more about this than me?

Reply:
yes it's no longer needed 🤗

Reply:
Exactly! https://github.com/okonet/lint-staged/releases/tag/v10.0.0